python/test/unit/language/test_core.py

```diff
@@ -3686,10 +3686,11 @@ def kernel(Out):
 # TODO: backend should be tested separately
 
 layouts = [
-    # MmaLayout(1, [1, 4], [1, 1], [0, 1]),
-    # MmaLayout((2, 0), [1, 4], [1, 1], [1, 1], [0, 1], [16, 8]),
-    # MmaLayout(1, [4, 1], [1, 1], [0, 1]),
-    # MmaLayout((2, 0), [4, 1], [1, 1], [1, 1], [0, 1], [16, 8]),
+    # Mma -> Mma layout conversion is not supported on Volta (version 1.0)
+    # MmaLayout((1, 0), [1, 4], [1, 1], [0, 1], [0,1], [16, 16]),
+    # MmaLayout((1, 0), [4, 1], [1, 1], [0, 1], [0, 1], [16, 16]),
+    MmaLayout((2, 0), [1, 4], [1, 1], [1, 1], [0, 1], [16, 8]),
```

Comment:
How about:
- a different number of total warps?
- a different minor version?

Reply:
Good question -- for some reason, I assumed mma -> mma layout conversion was supported on Hopper. But this is not the case.

Conversion of  `MmaLayout((3, 0), [1, 4], [1, 1], [1, 1], [0, 1], [16, 8, 16])` to `MmaLayout((3, 0), [4, 1], [1, 1], [1, 1], [0, 1], [16, 8, 16])` triggers this assertion:

https://github.com/openai/triton/blob/bd95d2d0de3a58c182192bb52e520550db8a48d0/lib/Conversion/TritonGPUToLLVM/ConvertLayoutOpToLLVM.cpp#L791

I'll investigate.